models/forms/class.XmlEditForm.php

```diff
@@ -0,0 +1,71 @@
+<?php
```

Comment:
Can we set namespace instead of using this old way to create classes?

Reply:
I don't think so. If I'm not mistaken,  for that we should refactor all old actions with namespaces